docs/private-networks/tutorials/permissioning/onchain.md

```diff
@@ -332,72 +383,30 @@ The migration logs the addresses of the Admin and Rules contracts.
     yarn start
     ```
 
```

Comment:
I think the extra spaces originally included in lines 386, 390-91, and 395-96 in this step 15 are required to group those lines into their corresponding sub-steps. See the [preview](https://hyperledger-besu--1153.org.readthedocs.build/en/1153/private-networks/tutorials/permissioning/onchain/#15-start-the-permissioning-management-dapp). Can you add those spaces back?